src/main/java/com/google/swarm/tokenization/common/Util.java

```diff
@@ -61,12 +62,17 @@
       DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss.SSSSSS");
   private static final String NESTED_SCHEMA_REGEX = ".*[^=]=(.*[^ ]), .*[^=]=(.*[^ ])";
   private static final String ALLOWED_FILE_EXTENSION = String.valueOf("csv");
+  //public static final String VALID_FILE_PATTERNS = "^(.(?!.*\\.ctl$|.*\\.dlp$|.*\\.xml$|.*\\.json$|.*parallel_composite_uploads.*$|.*\\.schema$|.*\\.temp$))*$";
+  public static final String VALID_FILE_PATTERNS = "([^\\s]+(\\.(?i)(dat|csv|txt))$)";
   public static final String ALLOWED_NOTIFICATION_EVENT_TYPE = String.valueOf("OBJECT_FINALIZE");
   public static String INSPECTED = "INSPECTED";
   public static String FAILED = "FAILED";
   public static TupleTag<Row> inspectData = new TupleTag<Row>() {};
   public static TupleTag<Row> auditData = new TupleTag<Row>() {};
   public static TupleTag<Row> errorData = new TupleTag<Row>() {};
+  public static TupleTag<KV<String, String>> readRowSuccess = new TupleTag<KV<String, String>>() {};
```

Comment:
needs documentation explaining what the key/value pairs are whenever using such generic structs